[flaky] test/e2e_new.TestTracingHeaders/TracingHeadersUsingUnorderedDeliveryWithMultipleTriggers/Assert/[Stable/MUST]received_event_has_traceparent_header

### Auto-generated issue tracking flakiness of test
* **Test name**: test/e2e_new.TestTracingHeaders/TracingHeadersUsingUnorderedDeliveryWithMultipleTriggers/Assert/[Stable/MUST]received_event_has_traceparent_header
* **Repository name**: eventing-kafka-broker

<!-------------End of issue body, Please don't edit below this line------------->
<!--[DONT_MODIFY_TEST_IDENTIFIER]'test/e2e_new.TestTracingHeaders/TracingHeadersUsingUnorderedDeliveryWithMultipleTriggers/Assert/[Stable/MUST]received_event_has_traceparent_header' in repo 'eventing-kafka-broker'[DONT_MODIFY_TEST_IDENTIFIER]-->

There is a connection refused error when sending the event to eventshub:
```
{"@timestamp":"2021-11-18T08:01:25.375Z","@version":"1","message":"[id: 0xe81127f9] RECEIVED: UDP [39105: /10.72.0.10:53], DatagramDnsResponse(from: /10.72.0.10:53, to: /0.0.0.0:53183, 39105, QUERY(0), NoError(0), RD AA RA)\n\tDefaultDnsQuestion(sink-veuqgzty.test-aqfxmpye.svc.cluster.local. IN A)\n\tDefaultDnsRawRecord(sink-veuqgzty.test-aqfxmpye.svc.cluster.local. 30 IN A 4B)","logger_name":"io.netty.resolver.dns.DnsNameResolver","thread_name":"vert.x-eventloop-thread-0","level":"DEBUG","level_value":10000}
{"@timestamp":"2021-11-18T08:01:26.322Z","@version":"1","message":"Sending event id=full-event-unordered subscriberURI=http://sink-veuqgzty.test-aqfxmpye.svc.cluster.local","logger_name":"dev.knative.eventing.kafka.broker.dispatcher.impl.http.WebClientCloudEventSender","thread_name":"vert.x-eventloop-thread-1","level":"DEBUG","level_value":10000,"id":"full-event-unordered","subscriberURI":"http://sink-veuqgzty.test-aqfxmpye.svc.cluster.local"}
{"@timestamp":"2021-11-18T08:01:26.323Z","@version":"1","message":"Sending event id=full-event-unordered subscriberURI=http://sink-veuqgzty.test-aqfxmpye.svc.cluster.local","logger_name":"dev.knative.eventing.kafka.broker.dispatcher.impl.http.WebClientCloudEventSender","thread_name":"vert.x-eventloop-thread-0","level":"DEBUG","level_value":10000,"id":"full-event-unordered","subscriberURI":"http://sink-veuqgzty.test-aqfxmpye.svc.cluster.local"}
{"@timestamp":"2021-11-18T08:01:26.372Z","@version":"1","message":"failed to send event to subscriber target=http://sink-veuqgzty.test-aqfxmpye.svc.cluster.local event=CloudEvent{id='full-event-unordered', source=http://example.com/source, type='com.example.FullEvent', datacontenttype='text/json', dataschema=http://example.com/schema, subject='topic', time=2020-03-21T12:34:56.780Z, data=BytesCloudEventData{value=[34, 104, 101, 108, 108, 111, 34]}, extensions={exstring=exstring, extime=2020-03-21T12:34:56.78Z, exint=42, exurl=http://example.com/source, exbinary=AAECAw==, exbool=true}}","logger_name":"dev.knative.eventing.kafka.broker.dispatcher.impl.http.WebClientCloudEventSender","thread_name":"vert.x-eventloop-thread-1","level":"ERROR","level_value":40000,"stack_trace":"io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: sink-veuqgzty.test-aqfxmpye.svc.cluster.local/10.72.7.188:80\nCaused by: java.net.ConnectException: Connection refused\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Unknown Source)\n\tat java.base/sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)\n\tat io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)\n\tat io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)\n\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\n\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\n\tat java.base/java.lang.Thread.run(Unknown Source)\n","target":"http://sink-veuqgzty.test-aqfxmpye.svc.cluster.local","event":{"data":{},"id":"full-event-unordered","source":"http://example.com/source","type":"com.example.FullEvent","subject":"topic","time":1584794096.780000000,"dataContentType":"text/json","dataSchema":"http://example.com/schema","specVersion":"V1","extensionNames":["exstring","extime","exint","exurl","exbinary","exbool"],"attributeNames":["datacontenttype","subject","specversion","id","source","time","type","dataschema"]}}
{"@timestamp":"2021-11-18T08:01:26.373Z","@version":"1","message":"Failed to send event to subscriber topic=knative-broker-test-aqfxmpye-broker-lfkavtdo partition=0 headers=[KafkaHeaderImpl{'traceparent': 00-acf9492836ac6a2f22d105621922c8f5-7927f60cf82a9945-01}, KafkaHeaderImpl{'ce_specversion': 1.0}, KafkaHeaderImpl{'ce_id': full-event-unordered}, KafkaHeaderImpl{'ce_source': http://example.com/source}, KafkaHeaderImpl{'ce_type': com.example.FullEvent}, KafkaHeaderImpl{'content-type': text/json}, KafkaHeaderImpl{'ce_dataschema': http://example.com/schema}, KafkaHeaderImpl{'ce_subject': topic}, KafkaHeaderImpl{'ce_time': 2020-03-21T12:34:56.78Z}, KafkaHeaderImpl{'ce_exstring': exstring}, KafkaHeaderImpl{'ce_extime': 2020-03-21T12:34:56.78Z}, KafkaHeaderImpl{'ce_exint': 42}, KafkaHeaderImpl{'ce_exurl': http://example.com/source}, KafkaHeaderImpl{'ce_exbinary': AAECAw==}, KafkaHeaderImpl{'ce_exbool': true}] offset=0 event=CloudEvent{id='full-event-unordered', source=http://example.com/source, type='com.example.FullEvent', datacontenttype='text/json', dataschema=http://example.com/schema, subject='topic', time=2020-03-21T12:34:56.780Z, data=BytesCloudEventData{value=[34, 104, 101, 108, 108, 111, 34]}, extensions={exstring=exstring, extime=2020-03-21T12:34:56.78Z, exint=42, exurl=http://example.com/source, exbinary=AAECAw==, exbool=true}}","logger_name":"dev.knative.eventing.kafka.broker.dispatcher.impl.RecordDispatcherImpl","thread_name":"vert.x-eventloop-thread-1","level":"ERROR","level_value":40000,"stack_trace":"io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: sink-veuqgzty.test-aqfxmpye.svc.cluster.local/10.72.7.188:80\nCaused by: java.net.ConnectException: Connection refused\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Unknown Source)\n\tat java.base/sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)\n\tat io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)\n\tat io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)\n\tat io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)\n\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)\n\tat io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)\n\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\n\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)\n\tat java.base/java.lang.Thread.run(Unknown Source)\n","topic":"knative-broker-test-aqfxmpye-broker-lfkavtdo","partition":0,"headers":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}],"offset":0,"event":{"data":{},"id":"full-event-unordered","source":"http://example.com/source","type":"com.example.FullEvent","subject":"topic","time":1584794096.780000000,"dataContentType":"text/json","dataSchema":"http://example.com/schema","specVersion":"V1","extensionNames":["exstring","extime","exint","exurl","exbinary","exbool"],"attributeNames":["datacontenttype","subject","specversion","id","source","time","type","dataschema"]}}
```

--- a/test/e2e_new/tracing_test.go
+++ b/test/e2e_new/tracing_test.go
@@ -86,7 +86,7 @@ func TracingHeadersUsingOrderedDelivery() *feature.Feature {
 	))
 	f.Setup("trigger is ready", trigger.IsReady(triggerName))
 
-	f.Setup("install source", eventshub.Install(
+	f.Requirement("install source", eventshub.Install(
 		sourceName,
 		eventshub.StartSenderToResource(broker.GVR(), brokerName),
 		eventshub.InputEvent(ev),
@@ -131,7 +131,7 @@ func TracingHeadersUsingUnorderedDelivery() *feature.Feature {
 	))
 	f.Setup("trigger is ready", trigger.IsReady(triggerName))
 
-	f.Setup("install source", eventshub.Install(
+	f.Requirement("install source", eventshub.Install(
 		sourceName,
 		eventshub.StartSenderToResource(broker.GVR(), brokerName),
 		eventshub.InputEvent(ev),
@@ -182,7 +182,7 @@ func TracingHeadersUsingUnorderedDeliveryWithMultipleTriggers() *feature.Feature
 	))
 	f.Setup("trigger is ready", trigger.IsReady(triggerAName))
 
-	f.Setup("install source", eventshub.Install(
+	f.Requirement("install source", eventshub.Install(
 		sourceName,
 		eventshub.StartSenderToResource(broker.GVR(), brokerName),
 		eventshub.InputEvent(ev),

